pom.xml

```
@@ -179,6 +182,16 @@
                 <skip.unit.tests>${skipTests}</skip.unit.tests>
             </properties>
         </profile>
+        <!-- Till https://github.com/powermock/powermock/issues/1094 -->
+        <profile>
+            <id>openjdk17</id>
+            <activation>
+                <jdk>[16,)</jdk>
```

Or maybe instead of enabling it only for versions that break we should enable it for all versions of Java where it is applicable i.e. 9+ - since that will also clean-up the associated warnings.
```suggestion
            <id>java9+</id>
            <activation>
                <jdk>[9,)</jdk>
```

---

I am against this, since currently JDK 14 is running with Travis and that does not needed this config. I do not know what is with jdk 15, but I can check it.
According to the Adoptium roadmap: https://adoptium.net/support.html 16 is still acceptable, this is why I left 16 there.

---

Agree "not needed", but still valid. I'm fine with us still getting warnings from tests on 9-15; they are obsolete anyway.